lib/string.js

```diff
@@ -72,6 +72,16 @@ internals.String.prototype._base = function (value, state, options) {
         }
     }
 
+    if (typeof value === 'string' &&
+        this._inner.replacements &&
+        options.convert) {
```

Comment:
Well then, move it back inside the if. Also can you squash your commits before I merge ?